tests/test_context.py

```diff
@@ -581,30 +598,16 @@ class ComponentWithComplexConditionalSlots(Component):
             {% load component_tags %}
             <div class="frontmatter-component">
                 <div class="title">{% slot "title" %}Title{% endslot %}</div>
-                {% if component_vars.is_filled.subtitle %}
+                {% if self.is_filled.subtitle %}
                     <div class="subtitle">{% slot "subtitle" %}Optional subtitle{% endslot %}</div>
-                {% elif component_vars.is_filled.alt_subtitle %}
+                {% elif self.is_filled.alt_subtitle %}
                     <div class="subtitle">{% slot "alt_subtitle" %}Why would you want this?{% endslot %}</div>
                 {% else %}
                 <div class="warning">Nothing filled!</div>
                 {% endif %}
             </div>
         """
 
-    class ComponentWithNegatedConditionalSlot(Component):
```

Comment:
Inlined this component here https://github.com/EmilStenstrom/django-components/pull/632/files#diff-0739130667ad94af9f1ac9b3ab331c42b6e4c5b0fe7bc42a3dbb47f900b7cfcaR741